wdl-lint/tests/lints/missing-blank-line-after-version/source.errors

```diff
@@ -6,15 +6,3 @@ note[VersionFormatting]: expected exactly one blank line after the version state
 6 │ │ workflow test {
   │ ╰^
 
-note[BlankLinesBetweenElements]: missing blank line
```

Comment:
This should not have been firing, as it's redundant with the prior diagnostic. The logic of `BlankLinesBetweenElements` was tweaked to prevent this.